receiver/filelogreceiver/README.md

```diff
@@ -221,6 +222,10 @@ Here is some of the information the file log receiver stores:
 
 Exactly how this information is serialized depends on the type of storage being used.
 
+### Archiving
+
+If `polls_to_archive` setting is used in conjunction with `storage` setting, file offsets older than three poll cycles are stored on disk rather than being discarded. This feature enables the receiver to remember file for a longer period and also aims to use limited amount of memory. 
```

Comment:
What happens when `polls_to_archive` is used but `storage` is not set? Should describe this in the docs too.